WinUIGallery/ControlPages/DesignGuidance/IconsPage.xaml

```diff
@@ -101,45 +102,67 @@
-                Text="How do I get the font?" />
-            <RichTextBlock>
-                <Paragraph>
-                    On Windows 11: There's nothing you need to do, the font comes with Windows.<LineBreak />
-                    On Windows 10: The Segoe Fluent Icons is not included by default on Windows 10. You can download it <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">here</Hyperlink>
-                    .<LineBreak />
-                </Paragraph>
-            </RichTextBlock>
+        <TextBlock Text="With the release of Windows 11, Segoe Fluent Icons Font is the recommended symbol icon font."/>
+        <labs:SettingsExpander Grid.Row="1"
+                  IsExpanded="False"
+                  Header="Instructions on how to use Segoe Fluent Icons">
+            <labs:SettingsExpander.Items>
+                <labs:SettingsCard ContentAlignment="Left" Padding="16,8">
+                    <RichTextBlock TextWrapping="Wrap">
+                        <Paragraph FontWeight="SemiBold">
+                            How to get the font
+                        </Paragraph>
+                        <Paragraph>
+                            On Windows 11: There's nothing you need to do, the font comes with Windows.<LineBreak />
+                            On Windows 10: The Segoe Fluent Icons is not included by default on Windows 10. You can download it
+                            <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">here</Hyperlink>.
+                        </Paragraph>
+                    </RichTextBlock>
+                </labs:SettingsCard>
+                <labs:SettingsCard ContentAlignment="Left" Padding="16,8">
+                    <StackPanel Orientation="Vertical" Spacing="8">
+                        <RichTextBlock TextWrapping="Wrap">
+                            <Paragraph FontWeight="SemiBold">
+                                How to use the font
+                            </Paragraph>
+                            <Paragraph>
+                                If you don't specify a <Run FontFamily="Consolas, monospace" FontWeight="Bold">FontFamily</Run>,
+                                or you specify a <Run FontFamily="Consolas, monospace" FontWeight="Bold">FontFamily</Run> that is not available on the system at runtime,
+                                the <Hyperlink NavigateUri="https://learn.microsoft.com/uwp/api/windows.ui.xaml.controls.fonticon">FontIcon</Hyperlink> falls back to the default font family defined by the
+                                <Run FontFamily="Consolas, monospace" FontWeight="Bold">SymbolThemeFontFamily</Run> theme resource.
+                            </Paragraph>
+                        </RichTextBlock>
+                        <controls:SampleCodePresenter
+                            SampleType="XAML"
+                            CodeSourceFile="Icons/FontIconSample1_xaml.txt">
```

Comment:
Might want to update this file as it still mentions `Segoe MDL2 Assets`?

<Button Name="ExampleButton1">
   <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xE790;"/>
</Button>

Reply:
Nope. Not the example. This example is explicitly using a font.

Reply:
It might be confusing though? It's a page about Segoe Fluent Icons and then a snippet is shown with Segoe MDL2 Assets?

Any other icon font would be a better example then - or maybe a oneliner explaining how to set an icon font explicitly?

Reply:
Yeah you're totally right. It was a smart and efficient solution to use this control, and I took an existing file. We can make a new example file for this specific place (?)

Reply:
Yeah, not sure where else it's used. Not mentioning Segoe MDL2 Assets at all in the WinUI 3 Gallery might not be a bad thing 😀!

Reply:
![image](https://github.com/microsoft/WinUI-Gallery/assets/65828559/14ba7029-d453-498e-950d-d343a989f810)
![image](https://github.com/microsoft/WinUI-Gallery/assets/65828559/d4cd94bf-7b81-45d8-bfb7-2052253b52fe)

Reply:
Question: I create and link to "Icons/FontIconSample2_xaml.txt", but _at first_ opening the page breaks :(
Then I looked at the properties of neighboring files and saw build property "content", so I set that to Sample2, and now the app works. Because the line `<Content Include="ControlPagesSampleCode\Icons\FontIconSample2_xaml.txt" />
` is added to the .csproj file. But it looks very out of place there...